File: model/RoomsRequested.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using static ETT.model.Room;

namespace ETT.model
{
    public class RoomsRequested
    {
        private int number { get; set; }
        private RoomType type { get; set; }
        public RoomsRequested() { }
        public RoomsRequested(int number, RoomType type)
        {
            this.number = number;
            this.type = type;
        }
        public int getNumber() {
            return number;
        }
        public void setNumber(int number) {
            this.number = number;
        }
        public RoomType getType() {
            return type;
        }
        public void setType(RoomType type) {
            this.type = type;
        }
    }
}

File: ExaminationTimetabling/src/main/java/com/timetabling/app/model/RoomsRequested.java
package com.timetabling.app.model;

import com.timetabling.app.enums.RoomType;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonIgnore;

public class RoomsRequested {

	@JsonProperty("Number")
	private Integer number;
	@JsonProperty("Type")
	private RoomType type;
	
	public RoomsRequested() {}

	@JsonIgnore
	public Integer getNumber() {
		return number;
	}

	@JsonIgnore
	public void setNumber(Integer number) {
		this.number = number;
	}

	@JsonIgnore
	public RoomType getType() {
		return type;
	}

	@JsonIgnore
	public void setType(RoomType type) {
		this.type = type;
	}
	
	public RoomsRequested(Builder builder) {
		this.number = builder.number;
		this.type = builder.type;
	}
	
	public static class Builder {
		private Integer number;
		private RoomType type;
		
		public Builder number(Integer number) {
			this.number = number;
			return this;
		}
		
		public Builder type(RoomType type) {
			this.type = type;
			return this;
		}
		
		public RoomsRequested build() {
			return new RoomsRequested(this);
		}
	}
	
}
